app/platform/forms.py

```diff
@@ -18,8 +22,27 @@ class CollectionForm(FlaskForm):
     name = StringField(
         "Collection name",
         validators=[DataRequired("Enter a collection name")],
-        filters=[lambda x: x.strip() if x else x],
+        filters=[strip_string_if_not_empty],
+        widget=GovTextInput(),
+    )
+    submit = SubmitField(widget=GovSubmitInput())
+
+
+class SectionForm(FlaskForm):
+    title = StringField(
+        "Section title",
+        validators=[DataRequired("Enter a section title")],
+        filters=[strip_string_if_not_empty],
+        widget=GovTextInput(),
+    )
+    submit = SubmitField(widget=GovSubmitInput())
+
+
+class FormForm(FlaskForm):
+    title = StringField(
+        "Form name",
```

Comment:
Did we lose the updates to these forms somehow? The question text should be the label here and then use the label rendering params in the templates please 🙏